app/src/main/res/values-nb-rNO/strings.xml

```diff
@@ -638,11 +638,24 @@
     <!-- This string is displayed as the title of the Bookmarks List, which contains the user's bookmarked websites. -->
     <string name="bookmarks_title">Bokmerker</string>
 
+    <!-- This string is displayed as the title of the Bookmarks List when the bookmarks list is empty. -->
+    <string name="bookmarks_empty">Bokmerkelisten din er tom</string>
+
+    <!-- This string is displayed under the bookmarks title in the bookmarks list. -->
+    <string name="bookmarks_description">Klikk på @-ikonet når du er på en side for å legge det til i bokmerker.</string>
```

Comment:
Is the "-" specific to some languages? I see that most of them don't use it but some do.